AdminUi/test/AdminUi.Tests.Integration/API/BaseApi.cs

```diff
@@ -30,9 +32,34 @@ protected async Task<HttpResponse<T>> Post<T>(string endpoint, RequestConfigurat
         return await ExecuteRequest<T>(Method.Post, endpoint, requestConfiguration);
     }
 
-    protected async Task<HttpResponse<T>> Delete<T>(string endpoint, RequestConfiguration requestConfiguration)
+    protected async Task<HttpResponse> Delete(string endpoint, RequestConfiguration requestConfiguration)
     {
-        return await ExecuteRequest<T>(Method.Delete, endpoint, requestConfiguration);
+        return await ExecuteRequest(Method.Delete, endpoint, requestConfiguration);
+    }
+
+    private async Task<HttpResponse> ExecuteRequest(Method method, string endpoint, RequestConfiguration requestConfiguration)
+    {
+        var request = new RestRequest(new PathString(ROUTE_PREFIX).Add(endpoint).Value, method);
+
+        if (!string.IsNullOrEmpty(requestConfiguration.Content))
+            request.AddBody(requestConfiguration.Content);
+
+        if (!string.IsNullOrEmpty(requestConfiguration.ContentType))
+            request.AddHeader("Content-Type", requestConfiguration.ContentType);
+
+        if (!string.IsNullOrEmpty(requestConfiguration.AcceptHeader))
+            request.AddHeader("Accept", requestConfiguration.AcceptHeader);
+
+        var response = await _client.ExecuteAsync(request);
+        var result = new HttpResponse();
+        if (!response.IsSuccessful)
+            result = JsonConvert.DeserializeObject<HttpResponse>(response.Content!);
```

Comment:
I don't understand this. Does this ever fill any property of HttpResponse at all?

Reply:
It serializes the error property.